easybuild/easyconfigs/t/TurboVNC/TurboVNC-3.0.1-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,68 @@
+easyblock = 'CMakeMake'
+
+name = 'TurboVNC'
+version = '3.0.1'
+
+homepage = 'https://www.turbovnc.org'
+description = """TurboVNC is a derivative of VNC (Virtual Network Computing) that is tuned to provide
+ peak performance for 3D and video workloads."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/TurboVNC/turbovnc/archive']
+sources = ['%(version)s.tar.gz']
+patches = ['TurboVNC-%(version)s_fix-libX11-libs.patch']
+checksums = [
+    'f0bd45e4e6f8cb8a52b5ccdef70fbc28ba8776591554c166db113eedc914dd86',  # 3.0.1.tar.gz
+    '11acc18ae10355c7c6816fad0a68ed55fbcb7a04efd30740c619a1d82ecbd4c2',  # TurboVNC-3.0.1_fix-libX11-libs.patch
+]
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+]
+
+dependencies = [
+    ('Java', '11', '', True),
+    ('X11', '20220504'),
+    ('pixman', '0.40.0'),
+    ('libjpeg-turbo', '2.1.3'),
+    ('Mesa', '22.0.3'),
+    ('libGLU', '9.0.2'),
+    ('libglvnd', '1.4.0'),
+    ('OpenSSL', '1.1', '', True),
+]
+
+osdependencies = [
+    ('pam-devel', 'libpam0g-dev'),
+]
+
+configopts = "-DTVNC_BUILDJAVA=0 -DTVNC_SYSTEMX11=1 "
+configopts += "-DX11_X11_LIB=$EBROOTX11/lib/libX11.%s " % SHLIB_EXT
+configopts += "-DX11_Xau_LIB=$EBROOTX11/lib/libXau.%s " % SHLIB_EXT
+configopts += "-DX11_SM_LIB=$EBROOTX11/lib/libSM.%s " % SHLIB_EXT
+configopts += "-DX11_ICE_LIB=$EBROOTX11/lib/libICE.%s " % SHLIB_EXT
+configopts += "-DX11_ICE_LIB=$EBROOTX11/lib/libXext.%s " % SHLIB_EXT
+configopts += "-DX11_ICE_LIB=$EBROOTX11/lib/libICE.%s " % SHLIB_EXT
```

Comment:
this doesn't look right to me, we're re-defining `X11_ICE_LIB` twice?

Reply:
@jfgrimm thanks for spotting it. I've removed the duplicate option.

Reply:
you've removed the last duplicate, but still have:
```
-DX11_ICE_LIB=.../libICE.%s
-DX11_ICE_LIB=.../libXext.%s
```
the second of which should probably be defining a different variable (`X11_XEXT_LIB` or something like that)

Reply:
@jfgrimm missed it, thanks. done